labgrid/driver/powerdriver.py

```diff
@@ -1,16 +1,16 @@
+import math
 import shlex
 import time
-import math
```

Comment:
Why do you have to change the import order of the existing imports?

Reply:
Tthe contribution-guideline says, that one should always use isort to sort the imports. It seems that the imports weren't sorted.

Reply:
outdated, changes reverted